<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="RadarGraphicView" name="graphicsView"/>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="FrameLeft" name="frameLeft">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="FrameBottom" name="frameBottom">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FrameLeft</class>
   <extends>QFrame</extends>
   <header>frameleft.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrameBottom</class>
   <extends>QFrame</extends>
   <header>framebottom.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RadarGraphicView</class>
   <extends>QGraphicsView</extends>
   <header>radargraphicview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
